config/routes.rb

```diff
@@ -2,7 +2,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   namespace :api do
     namespace :v1 do
-      resources :users, only: [:index, :show, :create, :update, :destroy]
+      resources :users, only: %i[index show create update destroy]
```

Comment:
do you need to use this explicitly? having a resource will implicitly allow you to have all these endpoints afaik